tools/packaging/release/update-repository-version.sh

```diff
@@ -153,19 +161,19 @@ bump_repo() {
 		local version_to_replace=${current_version}
 		if [ "${target_branch}" == "main" ] && [[ "${new_version}" =~ "rc" ]]; then
 			## remove the kata-deploy & kata-cleanup stable yaml files
-			git rm tools/packaging/kata-deploy/kata-deploy/base/kata-deploy-stable.yaml
-			git rm tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup-stable.yaml
+			git rm ${kata_deploy_stable_yaml}
```

Comment:
Nit: quote the variables

Where is @jodh-intel to say that??!! :)

Reply:
I made sure to quote all the vars I introduced.